src/Tes.RunnerCLI/Commands/ProcessLauncher.cs

```diff
@@ -3,58 +3,60 @@
 
 using System.Diagnostics;
 using System.Reflection;
-using System.Text;
 using Microsoft.Extensions.Logging;
+using Tes.Runner.Logs;
 using Tes.Runner.Transfer;
 
 namespace Tes.RunnerCLI.Commands
 {
     public class ProcessLauncher
     {
-        private readonly StringBuilder standardOut = new StringBuilder();
-        private readonly StringBuilder standardError = new StringBuilder();
+        const int LogWaitTimeout = 30;
```

Comment:
either use TimeSpan.FromSeconds(30), or need to rename to "LogWaitTimeoutSeconds"